tests/tests_pytorch/strategies/test_deepspeed_strategy.py

```diff
@@ -84,43 +84,6 @@ def automatic_optimization(self) -> bool:
         return False
 
 
-def test_deepspeed_lightning_module(tmpdir):
```

Comment:
we normally keep the tests and wrap them with a `deprecated_call()`, or move them to the test_remove_xxx

Reply:
done!